python/xorbits/deploy/kubernetes/tests/test_config.py

```diff
@@ -29,11 +30,18 @@
     XorbitsSupervisorsConfig,
     XorbitsWorkersConfig,
 )
+from ..external_storage.juicefs.config import (
+    PersistentVolumeClaimConfig,
+    PersistentVolumeConfig,
+    SecretConfig,
+)
 
 kubernetes = lazy_import("kubernetes")
+juicefs_available = "juicefs" in sp.getoutput("echo $(helm repo list)")
```

Comment:
What would happen when someone not install helm and want to run  juicefs UT? ``helm`` is not a required lib for xorbits.